apps/client/src/components/RegisterLookupSummary.js

```diff
@@ -25,6 +28,11 @@ const RegisterLookupSummary = ({
         <AddressLine address={address} />
         <EditButton
           onClick={() => {
+            trackEvent({
+              category: name.toLowerCase(),
+              action: actions.EDIT_ADDRESS,
+              name: address.postalCode.substring(0, 4),
+            });
```

Comment:
You're tracking the `postalCode` on the Edit button???

Also please watch the 🔤